### DISCUSSION TITLE
examples of complete projects?

### DISCUSSION THREAD

**Discussion by richardltc** [pocketbase/pocketbase#168] [Q&A] [answered]
Hi,

Are there any examples of complete projects for a JS front-end, ideally SvelteKit?

Thanks

Richard

**Comment by ganigeorgiev** [comment, marked as answer]
Except the Admin UI, there is no official example frontend project using PocketBase at the moment.

_After the Dart SDK, I may consider creating some sample todo/chat app, but we need to first stabilize a little the API before adding more examples and documentation._

Of course, if someone from the community wants to share their own project, feel free to do it in the comments and/or in the ["Show and tell"](https://github.com/pocketbase/pocketbase/discussions/categories/show-and-tell) discussions category.

**Reply by richardltc** [reply]
Thank you
